user-docs/latest/plugins/introduction-to-plugins.md

```diff
@@ -0,0 +1,13 @@
+# Introduction to plugins
+
+::: warning 🚧 This section of the user guide is a work in progress. Stay tuned!
+<br>
+:::
+
+Strapi is built around plugins of different kinds. There are the core plugins which are essential for your Strapi application to function, and therefore cannot be deactivated. But there are also other plugins, than can either be installed by default or not, to add more options and possibilities to your Strapi application.
+
+::: tip NOTE
+Core plugins which include the Content Manager and the Content-Types Builder are documented in their own sections of the user guide as they are quite specific (see [Introduction to the Content Manager](../content-types-builder/introduction-to-content-manager.md) and [Introduction to the Content-Types Builder](../content-types-builder/introduction-to-content-manager.md)). This Plugins section focuses on how to manage plugins in general and provides documentation for the others, non-core plugins.
+:::
+
+Plugins are managed from *General > Marketplace* and *General > Plugins* in the main navigation of the admin panel, for anything related to installation and deletion. Once installed, your plugins are available in *Plugins*, also accessible from the main navigation of the admin panel.
```

Comment:
You added an image in the intro of the Content-Type Builder plugin.
We can maybe add one here too?

Of the market place?